testsuite/cpptests/test_sort.h

```diff
@@ -35,18 +35,10 @@
 namespace nest
 {
 
-const bool
-is_sorted( std::vector< size_t >::const_iterator begin,
```

Comment:
where did this function go?

Reply:
We only sort `BlockVector`s, so this function is never used.